app/v1/hateoas/AmendHateoasBodies.scala

```diff
@@ -16,6 +16,7 @@
 
 package v1.hateoas
 
+import api.hateoas.HateoasLinks
```

Comment:
I don't think these are used anywhere

Reply:
This file are not used anywhere so I have deleted it.